bolt/src/main/java/com/slack/api/bolt/model/Installer.java

```diff
@@ -25,6 +25,22 @@
 
     void setTeamId(String teamId);
 
+    // ---------------------------------
+    // Org-level installation
+    // ---------------------------------
+
+    Boolean getIsEnterpriseInstall();
+
+    void setIsEnterpriseInstall(Boolean isEnterpriseInstall);
+
+    String getEnterpriseUrl();
```

Comment:
As @stevengill and I discussed, we decided to rename `orgDashboardGrantAccess` to `enterpriseUrl` for more clarity.